src/System.Private.CoreLib/shared/System/Globalization/DateTimeParse.cs

```diff
@@ -5246,6 +5246,7 @@ private static void Trace(string s)
             // Internal.Console.WriteLine(s);
         }
 
+        // for testing; do not make this readonly
```

Comment:
Is there a suppression that can be applied for future analyzer runs?

Reply:
We could add a pragma warning disable for IDE0044, but there are other analyzers that flag for this as well.  If this is inadvertently changed, then either a) tests should fail (at least based on the comment) or b) it won't have mattered :)